tools/FlatFlowMigrationCli/Constants.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace FlatFlowMigrationCli;
```

Comment:
Do we want to add `ProductConstructionService.` prefix like we do for the rest of the tools?

Reply:
I am not sure. I don't think we should just dogmatically add this namespace. For instance I would not add it for `ReproTool` neither.
I think it makes sense for `ProductConstructionService.Cli` because it literally controls the `ProductConstructionService` but for `FlatFlowMigration` and `ReproTool`, packages that do not leave this repo, I think they don't need to be namespaced.

We have this pattern elsewhere too actually - only things that leave the repo have the right namespace.